src/app/App.css

```
@@ -3,14 +3,256 @@
 	margin: 0;
 	box-sizing: border-box;
 }
-
 html {
 	font-family: sans-serif;
 	font-size: 18px;
 }
+main{
+	display: flex;
+}
+.sectionDates{
+	width: 25vw;
+	height: 100vh;
+	background-color: aliceblue;
+	padding: 40px 70px;
+}
+.sectionDates-header{
```

Cambiar a section-dates-header
```suggestion
.section-dates-header {
```